src/replit/database/database.py

```diff
@@ -241,8 +231,9 @@ async def delete(self, key: str) -> None:
         Raises:
             KeyError: Key does not exist
         """
+        body, content_type = encode_multipart_formdata({"key": key})
```

Comment:
Are there plans to use multipart for other requests than delete?

Reply:
Loose plans, yes. We already support multipart for set, migrating that codepath as well could be beneficial.

We currently have a gap for `get` that could be benefitted by some out-of-band encoding, but `POST` for "get" semantics isn't great. Unfortunately the urlspace for get is already pretty crowded, which is why this PR is just to permit people to delete the keys they've set but can't otherwise read.